linera-service/src/linera.rs

```diff
@@ -1494,13 +1511,16 @@ impl Runnable for Job {
                     _ => unreachable!(),
                 }
                 committee = Committee::new(validators, policy);
-                let certificate = chain_client.stage_new_committee(committee).await.unwrap();
-                context.update_wallet_from_client(&mut chain_client).await;
+                let result = chain_client.stage_new_committee(committee).await;
+                context.update_and_save_wallet(&mut chain_client).await;
+                let certificate = result.context("failed to stage committee")?;
                 info!("Staging committee:\n{:?}", certificate);
                 context.push_to_all_chains(&storage, &certificate).await;
 
                 // Remove the old committee.
-                let certificate = chain_client.finalize_committee().await.unwrap();
+                let result = chain_client.finalize_committee().await;
+                context.update_and_save_wallet(&mut chain_client).await;
+                let certificate = result.context("failed to finalize committee")?;
                 context.update_wallet_from_client(&mut chain_client).await;
```

Comment:
What's going on here?

Reply:
Right, this line can be removed. (Same below.)